docs/webaverse/nfts.md

```diff
@@ -38,4 +52,12 @@ Non-fungible means that it is not interchangable, so while you can trade NFTs, y
 
 By default, all transactions (buying, selling, trading, etc.) are done on the Webaverse sidechain. Using the Webaverse sidechain allows for extremely fast and completely free transactions.
 
-If you want to transfer your SILK or NFTs to the main Ethereum change, you can do so. The ability to move to the Ethereum blockchain also allows you to sell your NFTs on a marketplace like OpenSea.
+If you want to transfer your SILK or NFTs to the main Ethereum chain, you can do so from the ASSET item page on the website. Note: the user has to pay a gas transaction fee to do the transfer.
+
+![Transfer to mainnet](/img/xfermainnet.jpg)
+
+The ability to move to the Ethereum blockchain also allows you to sell your NFTs on a marketplace like OpenSea.
+
+### What happens when I mint a NFT?
+
+When you mint a file from the web interface or Discord bot it gets automatically uploaded to [IPFS](https://ipfs.io/) with data about the token uploaded to the Webaverse sidechain. This process takes 0 gas because its all on the [sidechain](https://ethereum.org/en/developers/docs/layer-2-scaling/#sidechains), a separate blockchain which runs in parallel to mainnet and operates independently. You can choose to then move it to mainnet afterwords, just pay the transaction fees.
```

Comment:
> afterwords
afterwards

Also it doesn't take "0 gas", it's more like gas is free.

Reply:
fixed